axis.py

```diff
@@ -582,7 +582,7 @@ def extract_bibliographic(self, element, ns):
             informal_name = format_tag.text
             seen_formats.append(informal_name)
             if informal_name not in self.DELIVERY_DATA_FOR_AXIS_FORMAT:
-                self.log.error("Unrecognized Axis format name for %s: %s" % (
+                self.log.warn("Unrecognized Axis format name for %s: %s" % (
```

Comment:
This isn't really an error, just a sign that you need to add the new format to DELIVERY_DATA_FOR_AXIS_FORMAT.